go.mod

```diff
@@ -163,6 +159,6 @@ require (
 )
 
 replace (
-	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
-	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
+	github.com/docker/distribution => github.com/docker/distribution v2.8.1+incompatible
```

Comment:
why is this replacement needed as it points to same location?

Reply:
maybe we can drop, was there before maybe to use that always